MPF.Processors/UmdImageCreator.cs

```diff
@@ -90,6 +90,10 @@ internal override List<OutputFile> GetOutputFiles(string baseFilename)
                             | OutputFileFlags.Artifact
                             | OutputFileFlags.Zippable,
                             "disc"),
+                        new($"{baseFilename}_drive.txt", OutputFileFlags.Required
```

Comment:
Not entirely sure if this is always output in UIC or if it's only that one branch version. To be safe, the Required flag can probably be removed. It can always be added if it's a guaranteed output.